### PR TITLE
python310Packages.psd-tools: Unbreak package

### PR DESCRIPTION
**Pull request by onny**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/aggdraw/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    owner = "pytroll";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-w3HlnsHYB0R+HZOXtzygC2RST3gllPI7SYtwSCVXhTU=";
+  };
+
+  checkInputs = [
+    numpy
+    pillow
+    pytestCheckHook
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/aggdraw/default.nix:24]:
```suggestion
    pytest
```

File: pkgs/development/python-modules/psd-tools/default.nix [hunk 2]
```diff
@@ -1,27 +1,46 @@
...
 
+  nativeBuildInputs = [ cython ];
+
   propagatedBuildInputs = [
+    aggdraw
     docopt
+    ipython
     pillow
+    scikitimage
   ] ++ lib.optionals isPy27 [ enum34 ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/psd-tools/default.nix:34]:
```suggestion
  ];
```

Comment (onny) [reply pkgs/development/python-modules/psd-tools/default.nix:34]: Good catch!

File: pkgs/development/python-modules/aggdraw/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/development/python-modules/aggdraw/default.nix:24]: But since we are using a custom checkPhase we are not using it.

Comment (onny) [reply pkgs/development/python-modules/aggdraw/default.nix:24]: Oh okay, good point

**Comment by onny** [comment]
Thank you @SuperSandro2000 for reviewing

**Pull request merged**